lib/internal/Magento/Framework/Filesystem/Glob.php

```diff
@@ -14,6 +14,7 @@
  */
 class Glob extends LaminasGlob
 {
+    protected static $cache = [];
```

Comment:
It looks like it should be private cache

```suggestion
    private static $cache = [];
```

Also, wonder if we should have a mechanism for removing the cache...
Will see after test results